lib/ex_unit/lib/ex_unit.ex

```diff
@@ -312,6 +314,19 @@ defmodule ExUnit do
     ExUnit.Runner.run(config, nil)
   end
 
+  @doc """
+  Allows users to configure functions to run after the test suite has finished.
```

Comment:
The first paragraph of the docs should always be a short summary. :) So I recommend adding a newline after this.

I also think that the "Allows users" doesn't add anything, as we could prefix most function documentations with "Allows users". :)

Reply:
Yeah, I have a lot of work on the documentation of this feature for sure.